Objective-C/CBLReplicator.mm

```diff
@@ -358,6 +358,73 @@ - (void) removeChangeListenerWithToken: (id<CBLListenerToken>)token {
     }
 }
 
+- (NSSet<NSString*>*) pendingDocumentIDs: (NSError**)error {
+    if (_config.replicatorType > 1) {
+        if (error)
+            *error = [NSError errorWithDomain: CBLErrorDomain
+                                         code: CBLErrorUnsupported
+                                     userInfo: @{NSLocalizedDescriptionKey: kCBLErrorMessagePullOnlyPendingDocIDs}];
+        return nil;
```

Comment:
I thought we are returning empty set instead of nil?

Reply:
Oops! i didn't see this, and you are right. it should be an empty set.

Reply:
https://issues.couchbase.com/browse/CBL-757